resources/views/users/index.blade.php

```diff
@@ -14,15 +14,15 @@
         </div>
     </div>
 
-    @if (count($admins) > 0)
+    @if ($admins->count())
         @include('users.partials._table_users', ['users' => $admins, 'role' => 'Admins'])
     @endif
 
-    @if (count($staff) > 0)
+    @if ($staff->count())
         @include('users.partials._table_users', ['users' => $staff, 'role' => 'Staff'])
     @endif
 
-    @if (count($contestants) > 0)
+    @if ($contestants->count())
```

Comment:
it's a collection and the collection technically contains an `items` property that is an empty array (if nothing inside), so I figured I'd use the `count()` method available on the collection, cause I was afraid anything else might do a false positive.